src/tests/ftest/util/mpio_test_base.py

```diff
@@ -41,16 +43,46 @@ def __init__(self, *args, **kwargs):
         super(LlnlMpi4pyHdf5, self).__init__(*args, **kwargs)
         self.hostfile_clients_slots = None
         self.mpio = None
+        self.daos_cmd = None
+        self.cont_uuid = None
 
     def setUp(self):
         super(LlnlMpi4pyHdf5, self).setUp()
 
+        # initialise daos_cmd
+        self.daos_cmd = DaosCommand(self.bin)
+
         # initialize a python pool object then create the underlying
         self.pool = TestPool(
             self.context, dmg_command=self.get_dmg_command())
         self.pool.get_params(self)
         self.pool.create()
 
+    def _create_cont(self, doas_cmd):
+        """Create a container.
+
+        Args:
+            daos_cmd (DaosCommand): doas command to issue the container
+                create
+
+        Returns:
+            str: UUID of the created container
+
+        """
+        cont_type = self.params.get("type", "/run/container/*")
+        result = self.daos_cmd.container_create(
+            pool=self.pool.uuid, svc=self.pool.svc_ranks,
+            cont_type=cont_type)
+
+        # Extract the container UUID from the daos container create output
+        cont_uuid = re.findall(
+            "created\s+container\s+([0-9a-f-]+)", result.stdout)
```

Comment:
(lint) '\s'. String constant might be missing an r prefix.
(lint) '\s'. String constant might be missing an r prefix.